### PR TITLE
661 - Add WeekView component

### PR DESCRIPTION
**Pull request by ericangeles**
**Explain the _details_ for making this change. What existing problem does the pull request solve?**
This PR adds WeekView component wrapper.

**Related github/jira issue (required)**:
Closes https://github.com/infor-design/enterprise-ng/issues/661

**Steps necessary to review your pull request (required)**:
- Pull this branch, build, and run the app
- Navigate to http://localhost:4200/ids-enterprise-ng-demo/week-view
- It should have a working weekview example

### CODE REVIEW

File: projects/ids-enterprise-ng/src/lib/week-view/soho-week-view.component.ts [hunk 1]
```diff
@@ -0,0 +1,390 @@
...
+      return;
+    }
+
+    if (this.updateRequired) {
+      this.updated();
+      this.updateRequired = false;
+    }
+  }
+
+  onWeekRenderedEvent(event: SohoWeekViewRenderEvent) {
```
Comment (pwpatton) [inline projects/ids-enterprise-ng/src/lib/week-view/soho-week-view.component.ts]: should these be private?

File: src/app/week-view/week-view.demo.html [hunk 2]
```diff
@@ -0,0 +1,2 @@
+<div soho-week-view
+></div>
```
Comment (pwpatton) [inline src/app/week-view/week-view.demo.html]: new line?

File: docs/CHANGELOG.md [hunk 3]
```diff
@@ -8,6 +8,10 @@
 - `[Datagrid]` Added `enableTooltips` Input to datagrid. `BTHH` ([#674](https://github.com/infor-design/enterprise-ng/pull/674))
 - `[FileUpload]` Fixed duplicate change event. `TJM` ([#672](https://github.com/infor-design/enterprise-ng/pull/672))
 
+### 6.3.0 Features
+
+- `[Week View]` - Added week view component wrapper. `EA` ([#661](https://github.com/infor-design/enterprise-ng/issues/661))
```
Comment (tmcconechy) [inline docs/CHANGELOG.md]: Just as a note in my other pending PR i also added this so may be a conflict to resolve.

**Comment by tmcconechy** [comment]
cc @ericangeles - still have plans to update this?

**Comment by ericangeles** [comment]
> cc @ericangeles - still have plans to update this?

Yep. Just stuck on something. But yeah.

**Comment by ericangeles** [comment]
@tmcconechy will close this PR.

**Pull request closed without merging**